internal/common/mqtt/mqttconnection_test.go

```diff
@@ -22,12 +22,46 @@ import (
 
 const Host = "broker.hivemq.com"
 const port = "1883"
+const InvalidHost = "www.broker"
+
+func initializeTest(Host string, clientID string) {
+	InitClientData()
+	StartMQTTClient(Host, clientID)
+}
 
 func TestStartMQTTClient(t *testing.T) {
 	t.Run("Success", func(t *testing.T) {
-		InitClientData()
-		StartMQTTClient(Host, "testClient")
+		initializeTest(Host, "testClient")
 		t.Log("Client connected")
+		client := clientData["testClient"]
+		message := Message{"appid", "Test data for testing"}
+		client.Publish(message, "home/livingroom")
+		StartMQTTClient(Host, "testClient")
```

Comment:
How about adding error check after `StartMQTTClient`? It returns an empty string when successful.

```suggestion
		ret = StartMQTTClient(Host, "testClient")
		if len(ret)>0 {
			t.Error("unexpected error")
		}
```

Reply:
done!